os-java-run/added/java-alternatives.sh

```diff
@@ -0,0 +1,9 @@
+set_java_alternatives() {
+  if [ "${USE_32_BIT_JVM^^}" = "TRUE" ]; then
```

Comment:
I think the goal of the jira was to automatically set this if the container size was less than a certain size.  Furthermore, there should probably be a way to disable using 32bit jvm if an user's application requires a native library that's unavailable (e.g. apr).

Reply:
+1 on setting this automatically, there's an example of this in the jenkins image

https://github.com/openshift/jenkins/blob/master/2/contrib/s2i/run#L126-L130

I'm not sure if it will be possible to determine reliably whether the libraries are present or not, they may be too far.

Reply:
I don't think we need to detect the presence of any native libs that may be required.  We just need a way for the user to disable using 32bit if they choose.  In other words, if they don't disable this feature, we use 32bit jvm if the container is not big enough for 64bit.